health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CreateQuestion.js

```
@@ -345,13 +357,15 @@ const CreateQuestion = ({ onSelect, className, level = 1, initialQuestionData, p
       {initialQuestionData
         ?.filter((i) => i.level === level && (i.parentId ? (i.parentId === parentId) : true) && (i.level <= 3) && (i.isActive === true))
         ?.map((field, index) => {
+          const questionNumber = getQuestionNumber(index, level, parentNumber);
           return (
             <div>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)